src/V3EmitCSyms.cpp

```diff
@@ -604,6 +639,10 @@ void EmitCSyms::emitSymImp() {
     puts("\n");
 
     puts("\n// FUNCTIONS\n");
+    puts(symClassName() + "::~" + symClassName() + "()\n");
```

Comment:
`test_regress` runs clean for me now.  But I'm not sure if having the destructor before the constructor is desirable.  If not, I'll need to understand more how the file splitting logic works more so that I can put it at the bottom of this method.

Reply:
Looks strange but I'm not going to worry about it.